scripts/prepare_automatic_backups.py

```diff
@@ -80,21 +95,33 @@ def update_cron_dict(cron_dict):
 
 
 def get_cron_dict():
+    """Converts yaml file content from yaml to dict type.
+
+    Returns:
+        dict(str, str). Returns yaml file content in dict format.
+    """
     return utils.dict_from_yaml(utils.get_file_contents(_CRON_YAML_FILE_NAME))
 
 
 def save_cron_dict(cron_dict):
+    """Converting dictionary content into yaml and saving into a yaml file.
+
+    Args:
+        cron_dict: dictionary. It is the content of updated yaml file.
+    """
     with open(_CRON_YAML_FILE_NAME, 'wt') as cron_yaml_file:
         cron_yaml_file.write(utils.yaml_from_dict(cron_dict))
 
 
 def update_yaml_files():
+    """Updates yaml files."""
```

Comment:
Don't just repeat the function name; explain what it's doing.

Reply:
done